static/index.html

```diff
@@ -52,20 +54,46 @@
             <th></th>
             <td>
               <div id="controls"
-                ><!--<button id="btnConnectWallet">Connect Wallet</button> --><button id="additionalDetails">
+                ><!--<button id="btnConnectWallet">Connect Wallet</button> -->
+                <button id="additionalDetails">
                   <div>Details</div
                   ><svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 96 960 960" width="24">
                     <path
                       d="M481.915 798Q497 798 507.5 787.584q10.5-10.415 10.5-25.5Q518 747 507.585 736.5q-10.416-10.5-25.5-10.5Q467 726 456.5 736.416q-10.5 10.415-10.5 25.5Q446 777 456.415 787.5q10.416 10.5 25.5 10.5ZM454 658h52q2-28.648 9.5-46.824T552 564q27-25 40.5-47.919 13.5-22.919 13.5-52.642Q606 413 569.754 383.5 533.507 354 484 354q-48 0-81 25.5T356 440l48 20q8-22 25-40t53-18q38 0 56 21t18 45q0 21-12 37t-30 33q-38 35-49 58t-11 62Zm26.174 298q-78.814 0-148.212-29.911-69.399-29.912-120.734-81.188-51.336-51.277-81.282-120.595Q100 654.988 100 576.174q0-79.07 29.97-148.694 29.971-69.623 81.348-121.126 51.378-51.502 120.594-80.928Q401.128 196 479.826 196q79.06 0 148.676 29.391 69.615 29.392 121.13 80.848 51.516 51.457 80.942 121.018Q860 496.819 860 575.909q0 79.091-29.391 148.149-29.392 69.059-80.835 120.496-51.443 51.436-120.987 81.441Q559.244 956 480.174 956ZM480 902q136.513 0 231.256-94.744Q806 712.513 806 576t-94.744-231.256Q616.513 250 480 250t-231.256 94.744Q154 439.487 154 576t94.744 231.256Q343.487 902 480 902Zm0-326Z"
                     />
-                  </svg> </button
-                ><button id="claimButton">
-                  <div class="claim-title">Claim</div
-                  ><svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 96 960 960" width="24" class="claim-icon">
+                  </svg>
+                </button>
+                <button id="claimButton">
```

Comment:
> If element tags were touching they should not be changed. 

I think it was touching because of my local prettier configuration, the repos prettier config had it spaced out like this and i don't think it'd introduce whitespace as the repo has been using the config for a while now

Reply:
https://github.com/ubiquity/pay.ubq.fi/pull/41/files#r1176457242

Reply:
https://github.com/ubiquity/pay.ubq.fi/pull/41/files#r1176457242